### PR TITLE
many: replace typographic quotes with ASCII

### PR DESCRIPTION
**Pull request by zyga**
Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

**Comment by mvo5** [comment]
This needs an update-pot run

File: daemon/api.go [hunk 1]
```diff
@@ -698,7 +698,7 @@ func sideloadSnap(c *Command, r *http.Request) Response {
 	contentType := r.Header.Get("Content-Type")
 
 	if strings.HasPrefix(contentType, "multipart/") {
-		// spec says POSTs to sideload snaps should be “a multipart file upload”
+		// spec says POSTs to sideload snaps should be "a multipart file upload"
```
Comment (chipaca) [inline daemon/api.go]: While we can't assume the *terminal* has unicode (and in particular, i think, utf8) support, I think we can assume the developers' editors and browser does. So comments and `.md` files are fine; output of `snap` (and `snappy`) is not.

Comment (zyga) [inline daemon/api.go]: I'd prefer to use ASCII consistently. It's not about compatibility (I think even windows will handle that nowadays) but about convenience of being able to copy-paste stuff around and see it work.

Comment (chipaca) [inline daemon/api.go]: alright

**Comment by chipaca** [comment]
:+1:

**Comment by niemeyer** [comment]
LGTM.. needs conflict solving.

**Pull request merged**